fiftyone/operators/types.py

```diff
@@ -1378,6 +1378,23 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
 
 
+class LazyFieldView(View):
+    """Displays a lazy text input which only apply input field changes on blur
+    or when user clicks the save button within the field.
+
+    .. note::
+
+        Must be used with :class:`String` or :class:`Number` properties.
+
+    Args:
+        disable_save_on_blur (None): prevents applying field changes on blur.
+        When set to True, user must click the save button to apply the changes.
```

Comment:
For Python docstrings, I believe Sphinx requires lines 2 and beyond of multiline parameter descriptions to be indented 4 spaces in order to be correctly rendered. For example:
https://github.com/voxel51/fiftyone/blob/48b050ba52de95dd7b8501d807485632f24ca172/fiftyone/core/dataset.py#L2864